### PR TITLE
[SM-4100] Create hadoop-client 2.7.7 bundle

### PR DESCRIPTION
**Pull request by johnpoth**
https://issues.apache.org/jira/browse/SM-4100

Thanks!

### CODE REVIEW

File: hadoop-client-2.7.7/pom.xml [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+        See the License for the specific language governing permissions and
+        limitations under the License.
+    -->
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.servicemix.bundles</groupId>
+        <artifactId>bundles-pom</artifactId>
+        <version>12</version>
```
Comment (jbonofre) [inline hadoop-client-2.7.7/pom.xml]: Please upgrade to bundles-pom 14.

File: hadoop-client-2.7.7/pom.xml [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+        <relativePath>../bundles-pom/pom.xml</relativePath>
+    </parent>
+
```
Comment (jbonofre) [inline hadoop-client-2.7.7/pom.xml]: Please add the scm section.

File: hadoop-client-2.7.7/pom.xml [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+    <groupId>org.apache.servicemix.bundles</groupId>
+    <artifactId>org.apache.servicemix.bundles.hadoop-client</artifactId>
+    <version>2.7.7_1-SNAPSHOT</version>
+    <packaging>bundle</packaging>
+    <name>Apache ServiceMix :: Bundles :: hadoop-client</name>
+    <description>This OSGi bundle wraps  ${pkgVersion} jar files.</description>
+
+    <scm>
+        <connection>scm:git:https://git-wip-us.apache.org/repos/asf/servicemix-bundles.git</connection>
```
Comment (jbonofre) [inline hadoop-client-2.7.7/pom.xml]: This one is incorrect. It's on gitbox now.

**Comment by johnpoth** [comment]
@jbonofre thanks for the review!

I tested with the Apache Camel  [Karaf itests](https://github.com/apache/camel/tree/master/tests/camel-itest-karaf) which is basic (testing that the feature installs correctly and all bundles are resolved). Thanks!

**Comment by jbonofre** [comment]
@johnpoth cool, I'm a bit concern by the number of optional imports as it can cause refresh. But it's something we can address later.

**Pull request merged**